jupyter_notebooks/Book_1_Intro_Williamson_2.ipynb

```diff
@@ -0,0 +1,380 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ec4b5fe",
+   "metadata": {},
+   "source": [
+    "# Notebook 1: Introduction with Shallow Water - Williamson 2 Test Case"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5371527b",
+   "metadata": {},
+   "source": [
+    "This notebook provides an introduction to Gusto. We will go through the example of the Williamson 2 test case to demonstrate how to set up the problem and run it. This comes from the paper: \\\n",
+    "David L. Williamson, John B. Drake, James J. Hack, Rüdiger Jakob, Paul N. Swarztrauber,\n",
+    "A standard test set for numerical approximations to the shallow water equations in spherical geometry,\n",
+    "Journal of Computational Physics,\n",
+    "Volume 102, Issue 1,\n",
+    "1992,\n",
+    "Pages 211-224,\n",
+    "ISSN 0021-9991,\n",
+    "https://doi.org/10.1016/S0021-9991(05)80016-6."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "664c0cba",
+   "metadata": {},
+   "source": [
+    "The Williamson 2 test case solves the shallow water equations. This corresponds to the following set of momentum and continuity equations, for a velocity $\\textbf{u}$ and free surface, $\\eta$:\n",
+    "$$\\textbf{u}_t + f \\textbf{u}^{\\perp} + g \\nabla \\eta + (\\textbf{u} \\cdot \\nabla) \\textbf{u} = 0$$\n",
+    "$$\\eta_t + H(\\nabla \\cdot \\textbf{u}) + \\nabla [\\textbf{u} (\\eta - b)] = 0$$\n",
```

Comment:
I have a very petty suggestion for how we write the shallow-water equations. I prefer to see the transport term (u.grad)u immediately after the time derivative in the momentum equation (so that you can easily see that together these are the material derivative). For the continuity equation, can we just write
`\\eta_t + \\nabla \\cdot [(\\eta - b) \\textbf{u}] = 0`?

Please feel free to ignore if you disagree!

Reply:
Yes, I agree with grouping the terms in that way and I also think we should try to be consistent with the names in the code - i.e. we use `D` for the prognostic variable, where `D=H+\eta-b`.